h/views.py

```diff
@@ -32,6 +32,20 @@ def _handle_exc(request):
         raise
 
 
+def _render_app(request, extra={}):
+    request.response.text = client_assets.render_app_html(
+      api_url=request.route_url('api'),
+      base_url=request.route_url('index'),
+      extra=extra,
+      ga_tracking_id=request.registry.settings.get('ga_tracking_id'),
+      register_url=request.route_url('register'),
```

Comment:
For now I guess I can live with this, but we probably shouldn't have two different ways of passing URLs into the client application. They should probably all live in the client settings object.